Windows/Translation-WPF/samples/TranslationServiceExample/app.config

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
+  <startup>
+    <supportedRuntime version="v4.0" sku=".NETFramework,Version=v4.6.1"/>
+  </startup>
+  <appSettings>
+    <add key="primaryKey" value="yourPrimaryKey"/>
+    <add key="luisAppID" value="yourLuisAppID"/>
+    <add key="luisSubscriptionID" value="yourLuisSubsrciptionID"/>
```

Comment:
>yourLuisSubsrciptionID [](start = 41, length = 22)

fix typo

Reply:
(also elsewhere)

---
In reply to: [193678261](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193678261) [](ancestors = 193678261)

Reply:
it is not used. Removed.

---
In reply to: [193680359](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/7#discussion_r193680359) [](ancestors = 193680359,193678261)